tests/upgrades/test_capsule.py

```diff
@@ -81,42 +83,27 @@ def test_pre_user_scenario_capsule_sync(self):
         :expectedresults: The repo/rpm should be synced to satellite
 
          """
-        _, env_name = hammer.hammer_determine_cv_and_env_from_ak(
-            self.activation_key, '1')
+        ak = entities.ActivationKey(organization=self.org_id).search(
+            query={'search': 'name={}'.format(self.activation_key)})[0]
+        env_id = ak.environment.id
+        env_name = ak.environment.read().name
```

Comment:
```suggestion
         ak_env = ak.environment.read()
```
Then, You can ```ak_env``` as:
```ak_env.id``` and ```ak_env.name```